dbt-tests-adapter/dbt/tests/adapter/incremental/fixtures.py

```diff
@@ -303,3 +303,34 @@
 
 from source_data
 """
+
+_MODELS__SRC_ARTISTS = """
+{{
+    config(
+        materialized='table',
+    )
+}}
+
+{% if var("version", 0) == 0 %}
+
+    select {{ dbt.current_timestamp() }} as inserted_at, 'john' as name
+
+{% else %}
+
+    -- add a non-zero version to the end of the command to get a different version:
+    -- --vars "{'version': 1}"
+    select {{ dbt.current_timestamp() }} as inserted_at, 'john' as Name, 'engineer' as "Job"
```

Comment:
This may be intentional, but should `Name` here match `name` above (casing)?

Reply:
was intentional to see if they would be seen differently or cause a issue (see bigquery) happy to make it lower case